### PR TITLE
changes in propert-service for errors

### PR DESCRIPTION
**Pull request by sairamya-egov**

### CODE REVIEW

File: municipal-services/land-services/pom.xml [hunk 1]
```diff
@@ -94,7 +98,7 @@
 		<dependency>
 			<groupId>org.postgresql</groupId>
 			<artifactId>postgresql</artifactId>
-			<version>42.2.2.jre7</version>
+			<version>42.2.18</version>
```
Comment (vinothrallapalli-eGov) [inline municipal-services/land-services/pom.xml]: Revert this change

File: municipal-services/sw-calculator/src/main/java/org/egov/swcalculation/service/SWCalculationServiceImpl.java [hunk 2]
```diff
@@ -102,7 +102,7 @@ public List<Calculation> getCalculation(CalculationReq request) {
 					request.getIsconnectionCalculation());
 			unsetSewerageConnection(calculations);
 		}
-		else if (request.getIsReconnectionRequest())	
+		if (request.getIsReconnectionRequest())
```
Comment (vinothrallapalli-eGov) [inline municipal-services/sw-calculator/src/main/java/org/egov/swcalculation/service/SWCalculationServiceImpl.java]: Why else condition is removed?

File: municipal-services/fsm/src/main/java/org/egov/fsm/service/VehicleTripService.java [hunk 3]
```diff
@@ -323,32 +323,4 @@ public void updateVehicleTrip(FSMRequest fsmRequest) {
 			}
 		}
 	}
-
```
Comment (vinothrallapalli-eGov) [inline municipal-services/fsm/src/main/java/org/egov/fsm/service/VehicleTripService.java:326]: Why this block of code is removed?

**Review by vinothrallapalli-eGov — CHANGES_REQUESTED** [review]

File: municipal-services/fsm/src/main/java/org/egov/fsm/service/VehicleTripService.java (hunk 3 above)
Comment (sairamya-egov) [reply municipal-services/fsm/src/main/java/org/egov/fsm/service/VehicleTripService.java:326]: Because it is repeated code.

File: municipal-services/land-services/pom.xml (hunk 1 above)
Comment (sairamya-egov) [reply municipal-services/land-services/pom.xml]: if the version is 42.2.2.jer7 it getting Jdbc template error.

Comment (sairamya-egov) [reply municipal-services/land-services/pom.xml]: Done vinoth

File: municipal-services/sw-calculator/src/main/java/org/egov/swcalculation/service/SWCalculationServiceImpl.java (hunk 2 above)
Comment (sairamya-egov) [reply municipal-services/sw-calculator/src/main/java/org/egov/swcalculation/service/SWCalculationServiceImpl.java]: it is also repeated code.I removed it once check it.

**Review by vinothrallapalli-eGov — APPROVED** [review]
